era5_to_int.py

```diff
@@ -73,6 +73,83 @@ def consider(self, field, xlvl, proj, hdate, slab, intfile):
             self.t = None
             self.td = None
 
+class RHDiags:
+    """ Implements the computation of RH (relative humidity in % at pressure level)
+    from ERA5 T (pressure level temperature) and Q (specific humidity at pressure level) fields.
+    """
+    def __init__(self, levels):
+        self.spechumd = {}
+        self.t        = {}
+        self.levels   = levels
+        
+
+    def liquidSaturationVaporMixRatio(self, t, p) :
+        t0 = 273.16
+        # From MPAS src/core_atmosphere/physics/mpas_atmphys_functions.F
```

Comment:
A block comment at the beginning of this function might be a better place to explain what this function computes and where the method of computation comes from.